src/lib/OpenAI/Auth/ManifestNoAuth.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Text.Json;
+
+namespace Microsoft.OpenApi.ApiManifest.OpenAI;
+
+public class ManifestNoAuth : BaseManifestAuth
+{
+    public ManifestNoAuth()
+    {
+        Type = "none";
+    }
+
+    internal static readonly FixedFieldMap<ManifestNoAuth> handlers = new()
+    {
+        { "type", (o,v) => {o.Type = v.GetString();  } },
```

Comment:
since the type is a field of the parent, what do you think about having protected methods that apply parent fields in essence to avoid duplication in all the derived types?